### PR TITLE
Release 0.15.1

### PR DESCRIPTION
**Pull request by andrewpmartinez**

### CODE REVIEW

**Comment by andrewpmartinez** [comment]
The TravisCI build is queued for linux. Been that way for a bit now. =(

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# Release 0.15.1
+
+* What's New:
```
Comment (plorenz) [inline doc/release-notes.md:3]: I'd probably say 'No new functionality introduced'. Can you use functionality as an adjective?

Comment (andrewpmartinez) [reply doc/release-notes.md:3]: Changed. Also I'm not sure.

**Comment by andrewpmartinez** [comment]
And we passed: https://jenkinstest.tools.netfoundry.io/blue/organizations/jenkins/ziti-smoke-test/detail/ziti-smoke-test/2868/pipeline/

**Pull request merged**